hub/models.py

```diff
@@ -1021,8 +1021,8 @@ class GeographyTypes(models.TextChoices):
         WARD = "WARD", "Ward"
         ADMIN_DISTRICT = "ADMIN_DISTRICT", "Council"
         PARLIAMENTARY_CONSTITUENCY = "PARLIAMENTARY_CONSTITUENCY", "Constituency"
-        PARLIAMENTARY_CONSTITUENCY_2025 = (
-            "PARLIAMENTARY_CONSTITUENCY_2025",
+        PARLIAMENTARY_CONSTITUENCY_2024 = (
```

Comment:
@conatus If we revert this line back to "2025", then I believe that will avoid the need for a migration?

Reply:
I think our options are: either go whole hog and write the migration, or restrict the changes to just the way Postcodes.io has the data and keep everything internal the same, to minimise change. Having an internal name for something that differs from its expression is going to be confusing, unless ring fenced.

Feels the former is the better solution and the latter is the for now solution if we don't think writing a migration is sensible,

What do you think?

Reply:
Whole hog IMO.